metadata/public/public_service_areas.json

```diff
@@ -0,0 +1,152 @@
+{
+    "@context": "",
+    "name": "public_service_areas",
+    "title": "Active Electric Utilities",
+    "description": "List of active electric utilities regulated by the Regulatory Commission of Alaska. On each community summary page, the Alaska Energy Data Gateway displays the utility serving that community and a map of its service area.\n",
+    "@id": "",
+    "resources": [
+        {
+            "@id": "",
+            "name": "public_service_areas",
+            "topics": [
+                "Electricity"
+            ],
+            "title": "Active Electric Utilities",
+            "path": "https://github.com/acep-aedg/aedg-data-pond/raw/refs/heads/main/data/public/public_service_areas.csv",
+            "description": "List of active electric utilities regulated by the Regulatory Commission of Alaska. On each community summary page, the Alaska Energy Data Gateway displays the utility serving that community and a map of its service area.\n",
+            "languages": [
+                "en-US"
+            ],
+            "keywords": [
+                "utility",
+                "electricity",
+                "grid",
+                "microgrids"
+            ],
+            "publicationDate": "2025-07-30",
+            "context": {
+                "title": "Alaska Energy Data Gateway v3.0",
+                "homepage": "https://akenergygateway.alaska.edu/",
+                "publisher": "Alaska Center for Energy and Power (ACEP) at the University of Alaska Fairbanks (UAF)",
+                "fundingAgency": "State of Alaska"
+            },
+            "spatial": {
+                "extent": {
+                    "name": "Alaska",
+                    "boundingBox": [
+                        -187.55,
+                        51.21,
+                        -130.0,
+                        71.35
+                    ],
+                    "crs": "OGC:CRS84"
+                }
+            },
+            "temporal": {
+                "referenceDate": "2025-08-01",
+                "timeseries": []
```

Comment:
The csv includes no temporal fields whatsoever. So I removed the timeseries metadata. We can revert this if it makes sense, but reading the OEMetadata docs, it really doesn't make sense in my opinion https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/oemetadata/latest/metadata_key_description.md